clap_complete/tests/snapshots/home/static/exhaustive/bash/.bashrc

```diff
@@ -542,6 +542,7 @@ _exhaustive() {
                     ;;
                 --other)
                     COMPREPLY=("${cur}")
+                    compopt -o nospace
```

Comment:
> Moved the test change to https://github.com/clap-rs/clap/commit/6e1fae93288d31107a93df2543aa440839a80a7b. I hope I fixed this correctly.

I wasn't asking for the snapshots to be moved but functional tests to be added.

Reply:
I completely misunderstood, sorry for bothering you.
I'll fix it.

Reply:
Locally `cargo test` is fine, but CI failed because completion output is different.
Even if we remove 'nosort', the sort orders can be different (https://github.com/clap-rs/clap/pull/5026/commits/35a9d8438e2aa0d396a807b420df7752ee957293).

Reply:
That is annoying.  nm then

Reply:
> Locally cargo test is fine, but CI failed because completion output is different.

Fixed by checking the existence of file names and/or directory names instead of comparing snapshot of the output.

Also, created files and directories to be completed in a temporary directory with [`snapbox::path::PathFixture::mutable_temp`](https://docs.rs/snapbox/latest/snapbox/path/struct.PathFixture.html#method.mutable_temp) so that these tests won't be affected by future changes to this project.